documentation/add/07-sqw_redesign.md

```diff
@@ -188,7 +209,7 @@ The children of **aProjection** class (e.g. **ortho_proj** class, defining ortho
 
 **axes_block** should be overloaded for every coordinate system defined by a projection, with possibility to overload plot methods for the convenience of the plotting in the particular coordinate system.
 
-Operations using projections and cut operations in particular result in the creation of a new SQW/DND object and are performed to the Image Pixels using data from the backing `PixelData`.
+Operations using projections and cut operations in particular result in the creation of a new `sqw/dnd` object and are performed to the Image Pixels using data from the backing `PixelData`.
```

Comment:
I think you want to repunctuate this as
'Operations using projections, and cut operations in particular, result in the creation of a new sqw/dnd object. They are performed on the Image pixels using...'
The end of the paragraph implies there is an available backing PixelData, which is only true for sqws. Are the projections done directly on the image voxels for independent DnDs? - this actually appears to be answered on L.249 but this paragraph should have the same info

Reply:
`PixelData` exist only on sqw.. I've added statement about image data cuts